tests/integration/cmd_push_test.go

```diff
@@ -159,21 +73,6 @@ var _ = Describe("odo push command tests", func() {
 	})
 
 	Context("when push command is executed", func() {
-		It("should not build when no changes are detected in the directory and build when a file change is detected", func() {
```

Comment:
Why are we removing this?

Reply:
Tests that have been removed without a comment have a devfile counterpart already present hence can be deleted